public/apps/configuration/panels/get-started.tsx

```diff
@@ -274,6 +290,7 @@ export function GetStarted(props: AppDependencies) {
           </EuiText>
         </EuiPanel>
       </div>
+      <EuiGlobalToastList toasts={toasts} toastLifeTimeMs={10000} dismissToast={removeToast} />
```

Comment:
qq, is this standard toast lifetime? What do you think about making it a global setting?

Reply:
I don't think there is a "standard" - I think it depends on the context of what is in the toast - in this case since the error message is a bit longer I think this is fine. I haven't looked into the other cases of this in our plugin, so I think that's outside of scope of this PR. Do you want to create a follow up issue to follow up on the consolidation?

Reply:
Okay. I don't think an issue is required if the toast message lifetime is not standard. If you'd still like to you can create one.